### DISCUSSION TITLE
Perguntas

### DISCUSSION THREAD

**Discussion by oeduardostein** [oeduardostein/oeduardostein#9] [Q&A] [answered]
Tenho uma dúvida sobre o seguinte código: console.log("Hello Word") ;. Percebi que a palavra "Word" está com "d" no final, mas acredito que o correto seria "Hello World". Alguém poderia me explicar se isso afeta o funcionamento do código e o que exatamente o comando console.log() faz no JavaScript? Obrigado!

**Comment by onicolasdelfino** [comment, marked as answer]
Sim, você está correto ao perceber que "Word" deveria ser "World" na mensagem de exemplo comum ("Hello World"). No entanto, o erro ortográfico de "Word" não afeta o funcionamento do código em si. O comando console.log("Hello Word") simplesmente imprime a string exatamente como está para o console do navegador ou ambiente de execução JavaScript.

O que o console.log() faz?
O método console.log() no JavaScript é usado para imprimir mensagens ou valores no console de depuração (geralmente o console do navegador ou de uma IDE). Ele é amplamente utilizado para verificar o comportamento do código ou visualizar valores enquanto o programa é executado.

Então, se você escreveu console.log("Hello Word"), o JavaScript imprimirá "Hello Word" no console, mesmo com o erro de digitação. Se você corrigir para console.log("Hello World"), ele imprimirá a string correta, "Hello World".

Em resumo:

O erro ortográfico não interfere no funcionamento do código, apenas altera a saída exibida no console.
O console.log() é uma ferramenta útil para debugging e visualização de informações durante o desenvolvimento.

**Reply by oeduardostein** [reply]
Muito Obrigado!
